docs/source/api/deliveryservice_stats.rst

```diff
@@ -37,140 +33,184 @@ Request Structure
-	+----------------------------+---------------+-----------------------------------------------------------------------------------------+
-	|``>average``                | float         |                                                                                         |
-	+----------------------------+---------------+-----------------------------------------------------------------------------------------+
-	|``>totalTransactions``      | int           |                                                                                         |
-	+----------------------------+---------------+-----------------------------------------------------------------------------------------+
-	|``series``                  | hash          | Series data                                                                             |
-	+----------------------------+---------------+-----------------------------------------------------------------------------------------+
-	|``>count``                  | int           |                                                                                         |
-	+----------------------------+---------------+-----------------------------------------------------------------------------------------+
-	|``>columns``                | array         |                                                                                         |
-	+----------------------------+---------------+-----------------------------------------------------------------------------------------+
-	|``>name``                   | string        |                                                                                         |
-	+----------------------------+---------------+-----------------------------------------------------------------------------------------+
-	|``>values``                 | array         |                                                                                         |
-	+----------------------------+---------------+-----------------------------------------------------------------------------------------+
-	|``>>time``                  | string        |                                                                                         |
-	+----------------------------+---------------+-----------------------------------------------------------------------------------------+
-	|``>>value``                 | float         |                                                                                         |
-	+----------------------------+---------------+-----------------------------------------------------------------------------------------+
-
-.. code-block:: json
+:series: An object containing the actual data series and information necessary for working with it.
+
+	:columns: This is an array of names of the columns of the data contained in the "values" array - should always be ``["time", "value"]``
```

Comment:
I am getting sum_count returned from the perl version instead of value

Reply:
Yeah, that's how it used to be in my version too. I decided to make it `value` instead, since `sum_count` is both puzzling to the user, as well as an exposure of implementation detail.

At the time it didn't occur to me that that'd be a breaking API change. But it would be...